gato/cli.py

```diff
@@ -0,0 +1,95 @@
+import asyncio
+
+from beaupy import confirm, prompt, select, select_multiple
+from gato import GatoGame
+from discord import Message, User
+from discord.ext.commands import Context
+from rich.console import Console
+
+
+console = Console()
+
+
+class LocalContext(Context):
+
+    author = User(state=None, data={
+                  'id': 1234, 'username': 'root', 'discriminator': '', 'avatar': ''})
+
+    def __init__(self):
+        pass
```

Comment:
hack the discord context so we don't need to initialise its stuff. 
All we need is for send to print to stdout instead.